crates/matrix-sdk-ui/src/encryption_sync/mod.rs

```diff
@@ -182,13 +173,30 @@ impl EncryptionSync {
         })
     }
 
+    /// Requests that the underlying sliding sync be stopped.
+    ///
+    /// This will unlock the cross-process lock, if taken.
     pub fn stop(&self) -> Result<(), Error> {
         // Stopping the sync loop will cause the next `next()` call to return `None`, so
         // this will also release the cross-process lock automatically.
         self.sliding_sync.stop_sync()?;
 
         Ok(())
     }
+
+    /// Request a reload of the internal caches used by this sync.
+    ///
+    /// This must be called every time the process running this loop was suspended and got back
```

Comment:
Only if another process is running in the background ;)

Reply:
Precised that, thanks.